docs/installation.md

```diff
@@ -22,6 +22,26 @@ $ nimble install owlkettle@#head
   pkg-config --libs gtk4
   ```
 
+### Properly setting up MSYS2 on Windows
+
+If you want to make sure you have the `pkg-config` in your system, 
+you need to install your MSYS2's environment toolchain.
+
+The `UCRT64` is the default environment you should use for any project, 
```

Comment:
For continuity with the previous sentence, you may want to mention that these are toolchains. That's beneficial, because then the reader immediately has *proof* in the sentence structure that you mean UCRT64 / CLANG64 when you speak of toolchains. A reader without any related knowledge would have to infer that.

So you could write this something like this:

There are 2 options worth mentioning for your toolchain:
1) The `UCRT64` toolchain, which is the recommended toolchain by default for most projects
2) The `CLANG64` toolchain, which is recommended for Nim

Reply:
> A reader without any related knowledge would have to infer that.

You are completely right!

Reply:
Fixed in: [give better instructions for users that don't have previous knowledge…](https://github.com/can-lehmann/owlkettle/pull/146/commits/08e74c9e6c33a26b45ade520cd94699f37a0b012)

Reply:
Resolved! I want Bitbuckets resolution interface now to be able to resolve these chains from my end as well!